pandas/core/categorical.py

```diff
@@ -118,6 +118,10 @@ def maybe_to_categorical(array):
 
 Assigning to `categories` is a inplace operation!
 
+Note
```

Comment:
This is supposed to be `Notes` (with an `s`). I'm not sure that the section is actually needed though.